test/views/ConfirmationPageSpec.scala

```diff
@@ -30,10 +30,6 @@ class ConfirmationPageSpec extends SpecBase with ViewTestHelper {
         titleShouldBeCorrect(view, "cf.cash-account.transactions.confirmation.statements")
       }
 
-      "backlink should take you back to request transactions" in new Setup {
```

Comment:
Is it worth us doing a test that says like Backlink should not exist rather just removing it? as I assume we're not getting back a filtered empty list here?